packages/dds/tree/src/tableSchema.ts

```diff
@@ -54,9 +54,7 @@ export namespace System_TableSchema {
 	 * Default type used for column and row "props" fields.
```

Comment:
```suggestion
	 * Default type used for column and row "props" fields.
	 * @privateRemarks
	 * Longer term, it would be better to simply omit "props" altogether by default.
	 * For now, this ensures that the user doesn't have to specify a "props" entry when initializing column/row nodes
	 * and ensures that they cannot set anything that might conflict with future evolutions of the schema.
```